cmd/sync.go

```diff
@@ -0,0 +1,13 @@
+package cmd
+
+import "github.com/spf13/cobra"
+
+var syncCmd = &cobra.Command{
```

Comment:
Should we remove this file?

Reply:
I don't think so, I think this organizes it pretty well by commands, you'll likely add more sync commands later.